Wikipedia/Code/YearInReviewSettingsViewController.swift

```diff
@@ -197,3 +198,14 @@ final class YearInReviewSettingsViewController: SubSettingsViewController {
     }
 
 }
+
+extension YearInReviewSettingsViewController: SavedArticleSlideDataDelegate {
+    func getSavedArticleSlideData(from startDate: Date, to endDate: Date) async -> SavedArticleSlideData {
+        await MainActor.run {
```

Comment:
Would be nice to find a way to avoid the duplicate code with WMFAppViewController+Extensions. Maybe a `SavedArticleSlideDataDelegate` default protocol implementation?

Reply:
I deduped a bit by moving the SavedArticleSlideDataDelegate conformance into MWKSavedPageList.